charts/telemetry-collector/values.yaml

```diff
@@ -0,0 +1,157 @@
+            "RRCConnEstabSucc": "rrc_connection_metrics",
+            "MMHoExeIntraReq": "handover_and_mobility_metrics",
+            "MMHoExeIntraSucc": "handover_and_mobility_metrics",
+            "RRCConnMean": "rrc_connection_metrics",
+            "RRCConnMax": "rrc_connection_metrics",
+            "SMPDUSessionSetupReq": "pdu_session_management_metrics",
+            "SMPDUSessionSetupReqSnssai": "pdu_session_management_metrics",
+            "SMPDUSessionSetupFail": "pdu_session_management_metrics",
+            "L1MSSRSRP": "rrc_measurements",
+            "MMHoPrepInterReq": "handover_and_mobility_metrics",
+            "MMHoPrepInterSucc": "handover_and_mobility_metrics",
+            "MMHoPrepInterFail": "handover_and_mobility_metrics",
+            "MMHoResAlloInterReq": "handover_and_mobility_metrics",
+            "MMHoResAlloInterSucc": "handover_and_mobility_metrics",
+            "MMHoResAlloInterFail": "handover_and_mobility_metrics",
+            "MMHoExeInterReq": "handover_and_mobility_metrics",
+            "AccDRBEstabAtt5QiSnssai": "radio_resource_utilization_metrics",
+            "DRBEstabSuccSnssai": "radio_resource_utilization_metrics",
+            "DRBEstabSucc5Qi": "radio_resource_utilization_metrics",
+            "DRBRelActNbr5QiSnssai": "radio_resource_utilization_metrics",
+            "DRBRelActNbrSnssai": "radio_resource_utilization_metrics",
+            "DRBRelActNbr5Qi": "radio_resource_utilization_metrics",
+            "MMHoExeInterReqTimeMean": "handover_and_mobility_metrics",
+            "MMHoExeInterReqTimeMax": "handover_and_mobility_metrics",
+            "AccGTPThpDl": "throughput_metrics",
+            "AccGTPThpDlQfiSnssaiPlmn": "throughput_metrics",
+            "AccGTPThpUl": "throughput_metrics",
+            "AccGTPThpUlQfiSnssaiPlmn": "throughput_metrics",
+            "AccDRBPdcpSduDelayDlQfiSnssaiPlmn": "radio_resource_utilization_metrics",
+            "AccDRBF1UPacketLossRateUlQoS": "radio_resource_utilization_metrics",
+            "AccDRBF1UPacketLossRateUlQfiSnssaiPlmn": "radio_resource_utilization_metrics",
+            "AccDRBF1UPacketLossRateUlSnssai": "radio_resource_utilization_metrics",
+            "AccDRBPdcpPacketDropRateDlQoS": "radio_resource_utilization_metrics",
+            "AccDRBPdcpPacketDropRateDlSnssai": "radio_resource_utilization_metrics",
+            "AccDRBPdcpPacketDropRateDlQfiSnssaiPlmn": "radio_resource_utilization_metrics",
+            "DRBPdcpSduDelayDlQoS": "radio_resource_utilization_metrics",
+            "DRBPdcpSduDelayDlSnssai": "radio_resource_utilization_metrics",
+            "DRBPdcpSduDelayDlDistQos": "radio_resource_utilization_metrics",
+            "DRBPdcpSduDelayDlDist": "radio_resource_utilization_metrics",
+            "DRBPdcpSduDelayDlDistSnssai": "radio_resource_utilization_metrics",
+            "AccDRBPdcpSduDelayDlDistQfiSnssaiPlmn": "radio_resource_utilization_metrics"
+          }
```

Comment:
We probably shouldn't use json here? It wouldn't be injected as a string now anyway, so the suggested change would be equivalent:

```suggestion
          buckets:
            RRCConnEstabAtt: "rrc_connection_metrics"
            RRCConnEstabSucc: "rrc_connection_metrics"
            MMHoExeIntraReq: "handover_and_mobility_metrics"
            MMHoExeIntraSucc: "handover_and_mobility_metrics"
            RRCConnMean: "rrc_connection_metrics"
            RRCConnMax: "rrc_connection_metrics"
            SMPDUSessionSetupReq: "pdu_session_management_metrics"
            SMPDUSessionSetupReqSnssai: "pdu_session_management_metrics"
            SMPDUSessionSetupFail: "pdu_session_management_metrics"
            L1MSSRSRP: "rrc_measurements"
            MMHoPrepInterReq: "handover_and_mobility_metrics"
            MMHoPrepInterSucc: "handover_and_mobility_metrics"
            MMHoPrepInterFail: "handover_and_mobility_metrics"
            MMHoResAlloInterReq: "handover_and_mobility_metrics"
            MMHoResAlloInterSucc: "handover_and_mobility_metrics"
            MMHoResAlloInterFail: "handover_and_mobility_metrics"
            MMHoExeInterReq: "handover_and_mobility_metrics"
            AccDRBEstabAtt5QiSnssai: "radio_resource_utilization_metrics"
            DRBEstabSuccSnssai: "radio_resource_utilization_metrics"
            DRBEstabSucc5Qi: "radio_resource_utilization_metrics"
            DRBRelActNbr5QiSnssai: "radio_resource_utilization_metrics"
            DRBRelActNbrSnssai: "radio_resource_utilization_metrics"
            DRBRelActNbr5Qi: "radio_resource_utilization_metrics"
            MMHoExeInterReqTimeMean: "handover_and_mobility_metrics"
            MMHoExeInterReqTimeMax: "handover_and_mobility_metrics"
            AccGTPThpDl: "throughput_metrics"
            AccGTPThpDlQfiSnssaiPlmn: "throughput_metrics"
            AccGTPThpUl: "throughput_metrics"
            AccGTPThpUlQfiSnssaiPlmn: "throughput_metrics"
            AccDRBPdcpSduDelayDlQfiSnssaiPlmn: "radio_resource_utilization_metrics"
            AccDRBF1UPacketLossRateUlQoS: "radio_resource_utilization_metrics"
            AccDRBF1UPacketLossRateUlQfiSnssaiPlmn: "radio_resource_utilization_metrics"
            AccDRBF1UPacketLossRateUlSnssai: "radio_resource_utilization_metrics"
            AccDRBPdcpPacketDropRateDlQoS: "radio_resource_utilization_metrics"
            AccDRBPdcpPacketDropRateDlSnssai: "radio_resource_utilization_metrics"
            AccDRBPdcpPacketDropRateDlQfiSnssaiPlmn: "radio_resource_utilization_metrics"
            DRBPdcpSduDelayDlQoS: "radio_resource_utilization_metrics"
            DRBPdcpSduDelayDlSnssai: "radio_resource_utilization_metrics"
            DRBPdcpSduDelayDlDistQos: "radio_resource_utilization_metrics"
            DRBPdcpSduDelayDlDist: "radio_resource_utilization_metrics"
            DRBPdcpSduDelayDlDistSnssai: "radio_resource_utilization_metrics"
            AccDRBPdcpSduDelayDlDistQfiSnssaiPlmn: "radio_resource_utilization_metrics"
```